rekuho/src/lib/Portion.svelte

```diff
@@ -8,25 +8,29 @@
   import type { CaretPosition } from "./core"
   import { state } from "./core"
 
-  import type { Theme } from "./theme"
+  import type { Config } from "./config"
   import { measureOnChild as fontMeasure, fontToString } from "./font"
   import type { Vector2 } from "./linearAlgebra"
-  import type { KeyInput } from "./rpc"
-  import { keyboardToKeyInput } from "./event"
+  import type { KeyInput, MouseWheel } from "./rpc"
+  import { keyboardToKeyInput, getModifiers } from "./event"
 
-  export let theme: Theme
+  type pixels = number
+  type line = number
 
-  let width: number
-  let height: number
+  export let config: Config
+
+  let width: pixels
+  let height: pixels
 
   const gutterWidth = 50 // TODO: maybe should be part of theme, minimum value?
   let input: HTMLTextAreaElement
   let container: HTMLElement
 
   const dispatch = createEventDispatcher<{
-    resize: [number, number]
+    resize: [pixels, pixels]
```

Comment:
For consistency, shouldn't we also make this an object with `width` and `height` or sth? Although I guess especially here with the pixels typealias this works 🤷🏻

Reply:
[x, y]